napps/ofcore/main.py

```diff
@@ -81,7 +81,7 @@ def handle_raw_message_in(self, event):
         self.add_to_msg_in_buffer(new_event)
 
     @ListenTo('KycoMessageInHello')
```

Comment:
This should be listen_to right ? And if so, why is passing on tests ? The import should failed, right ?